browser/binance/binance_util.cc

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/binance/binance_util.h"
+
+#include "chrome/browser/profiles/profile.h"
+#include "components/country_codes/country_codes.h"
+#include "brave/browser/binance/static_values.h"
```

Comment:
`browser/BUILD.gn` needs dep of `components/country_codes`

Reply:
added